### PR TITLE
Adds customizable blur. 

### PR DESCRIPTION
**Pull request by doggybootsy**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,25 +13,32 @@ const filterActivities = (a, i) => {
...
+	constructor() {
+		super();
+	}
+
     async startPlugin() {
+        this.reloadBlur = this.reloadBlur
+        // Blur
+		const { get, set } = this.settings;
+		if (!get('blur-album-scale')) set('blur-album-scale', 1);
+		const blurAlbumAmount = get('blur-album-scale');
```
Comment (Crenshaw1312) [inline index.js:25]: Should just `use this.settings.get()` since breaking this.settings apart for a few lines isn't worth it

File: Settings.jsx [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+const { SwitchItem, SliderInput } = require('powercord/components/settings')
+const path = require('path')
 module.exports = class Settings extends React.PureComponent {
+
+	constructor(props) {
+		super(props);
+	}
+
     render() {
+		const { getSetting, toggleSetting, updateSetting } = this.props
```
Comment (Crenshaw1312) [inline Settings.jsx:12]: If you're going to do this, I reccomened making the rest line up with it instead of just the on switch.

**Review by Crenshaw1312 — CHANGES_REQUESTED** [review]
Everything looks good, but please fix constancy, when you break apart something, make it so either everything below uses the same formatting, or just don't break it apart.

File: index.js (hunk 1 above)
Comment (doggybootsy) [reply index.js:25]: i tested and it didnt work
the if sets the default(if it hasnt already been set and just gets it), and the second const gets a const name.
but removing the set still works

File: Settings.jsx (hunk 2 above)
Comment (doggybootsy) [reply Settings.jsx:12]: i dont get what you mean.

**Comment by Crenshaw1312** [comment]
Look, don't take this poorly, but I'm going to merge, try to do what I mean myself bc I'm shit at explaining and I've had a long day and really need to program something to take my mind off things.

**Pull request merged**